tests/test_filter.py

```diff
@@ -441,3 +508,46 @@ def test_viscosity_filter(vector_grid_type_and_input_ds, filter_args):
             filter = Filter(
                 grid_type=grid_type, grid_vars=grid_vars_missing, **filter_args
             )
+
+
+@pytest.mark.parametrize(
+    "filter_args",
+    [dict(filter_scale=4.0, dx_min=1.0, n_steps=0, filter_shape=FilterShape.GAUSSIAN)],
+)
+@pytest.mark.parametrize(
+    "n_iterations",
+    [2, 3, 4],
+)
```

Comment:
Do you need to re-parameterize? I think it will work if you delete these 4 lines because they are just repeated from above.

Reply:
I deleted the lines (the repeated ones, not the first ones) and it threw an error, so I guess I need the lines. I don't really know much about how `pytest` works.

Reply:
Oh ok, my bad! I'm still learning about `pytest` too, so now we know one more thing about it. 😃